tests/common.go

```diff
@@ -78,6 +79,7 @@ import (
 	"github.com/portworx/torpedo/pkg/log"
 
 	context1 "context"
+
```

Comment:
Same here.

Reply:
this is comming by default after saving the file